examples/medplum-provider/src/components/messages/ChatList.tsx

```diff
@@ -0,0 +1,34 @@
+import { Stack } from '@mantine/core';
+import { Communication, Patient, Reference } from '@medplum/fhirtypes';
+import { JSX } from 'react';
+import { ChatListItem } from './ChatListItem';
+
+interface ChatListProps {
+  communications: Communication[];
+  selectedPatient: Reference<Patient>;
+  onClick: (patient: Reference<Patient>) => void;
+}
+
+export const ChatList = (props: ChatListProps): JSX.Element => {
+  const { communications, selectedPatient, onClick } = props;
+
+  return (
+    <Stack gap={0} p="xs">
+      {communications.map((communication: Communication) => {
+        const patient = communication.subject as Reference<Patient>;
+        const _isSelected = selectedPatient === patient;
```

Comment:
Would it be better to compare `.id`? Or are we reasonably sure the `Patient` objects will always be the same?